SQL/database_changelog.md

```diff
@@ -15,6 +15,13 @@ INSERT INTO `SS13_schema_revision` (`major`, `minor`) VALUES (5, 27);
 
 In any query remember to add a prefix to the table names if you use one.
 -----------------------------------------------------
+Version 5.28, TODO 2024, by Ghommie.
+Adds an entries column to achievements for the score/progress subtype that derivates their value from it.
+
+```sql
+ALTER TABLE `achievements` ADD COLUMN `entries` TEXT(8192) NULL AFTER `value`;
```

Comment:
this strikes me as a really bad idea, using this column as a list inherently limits its size.
my gut is it would be better to have a new table that stores each catch as a row, so we don't have weird hidden limits.

Reply:
That'd probably be for the best.

Reply:
actually, I'm not sure how and if making a new table that would work, since we need to associate entries with ckeys, and if that's done by having a ton of columns, it's quickly going to be disastrous.

Anyway, I've just realized that the TEXT datatype is deprecated as fuck, and that there's a JSON data type (basically another keyword LONGTEXT in mariaDB, which is realistically pretty much impossible to overflow even if we insert every typepath in the entire codebase). I'll be using that, also because I need to store an identifier key needed to version check the list in case typepaths are changed.